Apps/phwinrm/parse_callbacks.py

```diff
@@ -91,22 +91,34 @@ def list_processes(action_result, response):
         )
 
     output = response.std_out
-    lines = output.splitlines()
-    for line in lines[2:]:
-        process = {}
+    lines = list(filter(lambda x: not x.startswith('-----'), filter(lambda x:x, map(lambda x:x.decode('utf8').strip(), output.splitlines()))))
+    header = list(map(lambda x:x.lower(), lines[0].split() + [f"unknown_column_{x}" for x in range(0, 10)]))
```

Comment:
can we rewrite these with just list comprehensions? all the anonymous functions make it harder to parse what this is doing